src/test/java/core/basesyntax/dao/FileReaderCsvImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileReaderCsvImplTest {
+    public static final String INPUT_FILE = "src/test/java/resources/TestInput.csv";
+    private FruitFileReader fileReader;
+
+    @BeforeEach
+    void setUp() {
+        fileReader = new FileReaderCsvImpl();
+    }
+
+    @Test
+    void testFileReaderNotCsv_NotOk() {
+        assertThrows(IllegalArgumentException.class,() -> fileReader.read("Test.txt"));
+    }
+
+    @Test
+    void testFileReaderThrowsException_NotOk() {
+        assertThrows(RuntimeException.class,() -> fileReader.read(""));
+    }

```

Comment:
It’s not really informative. Modify the method name or add a message. Move path to the constant fields